### ISSUE TITLE
error TS2694: Namespace 'winston' has no exported member 'DefaulLevels'.   

### ISSUE THREAD

**Issue by gopalanand333** [SAP/cloud-sdk-cli#72]
While running npm run deploy, I'm getting the following error:
`>nest build
node_modules/@sap-cloud-sdk/util/dist/logger/cloud-sdk-logger.d.ts:5:47 - error TS2315: Type 'Logger' is not generic.
5 export declare const cloudSdkExceptionLogger: Logger<import("winston").DefaulLevels>;
                                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
node_modules/@sap-cloud-sdk/util/dist/logger/cloud-sdk-logger.d.ts:5:72 - error TS2694: Namespace 'winston' has no exported member 'DefaulLevels'.   

5 export declare const cloudSdkExceptionLogger: Logger<import("winston").DefaulLevels>;
                                                                         ~~~~~~~~~~~~
`

**To Reproduce**
Run the command:
npm run start:dev or 
npm run deploy

**Screenshots**
[bug.txt](https://github.com/SAP/cloud-sdk-cli/files/4850401/bug.txt)


If applicable, add screenshots to help explain your problem.

**Used Versions:**
@sap-cloud-sdk/cli/0.1.9 win32-x64 node-v12.16.1

**Comment by marikaner** [comment]
Thank you for reporting this - I just ran into the same issue. I will check that.

**Comment by marikaner** [comment]
@gopalanand333 It seems that one of our dependencies has an issue with its types. As a quickfix you could install winston@3.2.2, I created a ticket here: https://github.com/winstonjs/winston/issues/1822.

**Comment by gopalanand333** [comment]
Hi @marikaner,
I have modified package.json of @sap-cloud-sdk with Winston version  "winston": "3.3.2". 
The version 3.2.2 is not available at npm. 
The project is running now.

**Comment by marikaner** [comment]
Yes, I mistyped.

**Comment by marikaner** [comment]
Hey @gopalanand333,

it seems that winston changed their API between patch versions. Version 3.3.3 is supposedly the final one, I will update winston to this version then the issue should be fixed. Link to the winston issue: https://github.com/winstonjs/winston/issues/1822#event-3508252985.

**Comment by nocheintobi** [comment]
Hi, any news on this one?

**Comment by marikaner** [comment]
Yes, we will release a new version today.

**Comment by marikaner** [comment]
This issue should be resolved in version 1.24.0. Feel free to open a new issue if this is not the case for some reason.
